src/workerd/server/workerd.capnp

```diff
@@ -630,6 +630,12 @@ struct Worker {
 
   moduleFallback @13 :Text;
 
+  # Tail worker configuration
+  logging :union {
```

Comment:
This could have just been:

```
tails @14 :List(ServiceDesignator);
```

The union adds unnecessarily complexity.

Is anyone using this yet? Can we change it?

Please remember to tag me on changes to capnp APIs.